docs(storybook): docs cleanup (#6495)
adobe/spectrum-web-components: 2nd-gen/packages/swc/.storybook/learn-about-swc/get-started.mdx

Reviewer: ```suggestion
Import the SWC stylesheet in your app setup from either your TypeScript or JavaScript entry or your root CSS file.
```

Reviewer: ```suggestion
Apply `swc-theme` and one color class on a root element. Nested SWC components inherit it. The theme defaults to light if no class is specified.
```

> This will load the Spectrum 2 tokens, typography, and base app styles.

Reviewer: ```suggestion
This loads Spectrum 2 tokens, typography, and base app styles.
```

</html>
```

> - `swc-theme--light`: light color scheme.
> - `swc-theme--dark`: dark color scheme.
> - `swc-theme--adaptive`: follows the user's OS setting.

Reviewer: ```suggestion
- `swc-theme--light`: light color scheme
- `swc-theme--dark`: dark color scheme
- `swc-theme--adaptive`: follows the user's OS setting
```

## Where to look next

- **Components**: individual building blocks: badge, button, avatar, progress-circle, and more.
- **Patterns**: higher-level UI flows composed of components.
- **Guides**: deeper setup: theme color schemes, fonts, customization, accessibility.
- **[Support and compatibility](/docs/resources-support-and-compatibility--docs)**: versioning, APIs, and browser support for `@adobe/spectrum-wc`.

> - **[Changelog](/docs/resources-changelog--docs)**: release notes for `@adobe/spectrum-wc`.

Reviewer: ```suggestion
- **Components**: individual building blocks, like Badge, Button, Avatar, Progress Circle, and more
- **Patterns**: higher-level UI flows composed of components
- **Guides**: deeper dives about theme color schemes, fonts, customization, and accessibility
- **[Support and compatibility](/docs/resources-support-and-compatibility--docs)**: versioning, APIs, and browser support for `@adobe/spectrum-wc`
- **[Changelog](/docs/resources-changelog--docs)**: release notes for `@adobe/spectrum-wc`
```
